language/vm/src/deserializer.rs

```diff
@@ -178,6 +176,7 @@ fn read_table(cursor: &mut Cursor<&[u8]>) -> BinaryLoaderResult<Table> {
 
 fn read_table_contents(cursor: &mut Cursor<&[u8]>, n: usize) -> BinaryLoaderResult<Vec<u8>> {
     let mut bytes = vec![];
+    // TODO: can we rewrite this in a better way?
```

Comment:
do we need this TODO?

Reply:
No harm to keep it. There must be a way to read the whole blob without making so many IO calls.